Sources/PrimerSDK/Classes/Extensions & Utilities/BundleExtension.swift

```diff
@@ -32,7 +32,13 @@ internal extension Bundle {
         if let reactNativeVersion = Primer.shared.integrationOptions?.reactNativeVersion {
             return reactNativeVersion
         }
-        return Bundle.primerFramework.infoDictionary?["CFBundleShortVersionString"] as? String
+        let version = Bundle.primerFramework.infoDictionary?["CFBundleShortVersionString"] as? String
+        
+        if version != "2.16.5" {
+            return "2.16.5"
+        } else {
+            return version
+        }
     }
```

Comment:
what's your plan to make sure this is always up to date?

Reply:
First I want to see if the fix worked. If it worked, then this will be removed.